tools/pipelines/build-client.yml

```diff
@@ -139,3 +139,25 @@ extends:
     - syncpack:versions
     - check:versions
     - prettier
+    additionalBuildSteps:
+      - task: Bash@3
+        displayName: Inject devtools telemetry logger token
+        inputs:
+          script: |
+            echo Generating .env
+            echo "DEVTOOLS_TELEMETRY_TOKEN=$(devtools-telemetry-key)" >> ./packages/tools/devtools/devtools-browser-extension/.env
+            cat ./packages/tools/devtools/devtools-browser-extension/.env
+            
+      - task: Npm@1
+        displayName: Build devtools
+        inputs:
+          command: 'custom'
+          workingDir: ./packages/tools/devtools/devtools-browser-extension/
+          customCommand: 'run webpack'
+      
+      - task: PublishPipelineArtifact@1
+        displayName: Publish Artifact - Test Files
```

Comment:
Probably best to call this something like "Publish Artifact - Devtools Browser Extension"